src/flag_gems/__init__.py

```diff
@@ -67,6 +67,7 @@ def enable(lib=aten_lib):
     lib.impl("triu", triu, "CUDA")
     lib.impl("var_mean.correction", var_mean, "CUDA")
     lib.impl("linalg_vector_norm", vector_norm, "CUDA")
+    lib.impl("where.self", where, "CUDA")
```

Comment:
there are interfaces like where.Scalar, where.ScalarOther and so on. I suggest implementing them all.

Reply:
yes，same as above comment